src/main/java/racingcar/utils/NumberGenerator.java

```diff
@@ -1,4 +1,4 @@
-package racingcar.service;
+package racingcar.utils;
 
 public interface NumberGenerator {
 	public int getNumber();
```

Comment:
```suggestion
	public int generateNumber();
```
get~ 는 보편적으로 getter에만 쓰이기 때문에 이 경우 다른 네이밍을 쓰는게 좋을 것 같습니다 😉